src/commands/sign-and-publish.yml

```diff
@@ -49,4 +49,10 @@ steps:
 
         npm publish $TARPATH --tag <<parameters.tag>> --access <<parameters.access>>
 
+        # Wait for the version to become available on NPM
```

Comment:
I copied this over to my changes: https://github.com/forcedotcom/npm-release-management-orb/pull/11/files#diff-942e6fc836147fbac80e84de03aad97eR48